storage/innobase/xtrabackup/src/backup_copy.cc

```diff
@@ -2584,9 +2585,7 @@ bool decrypt_decompress_file(const char *filepath, uint thread_n) {
   if (ends_with(filepath, ".xbcrypt") && opt_decrypt) {
     cmd << " | xbcrypt --decrypt --encrypt-algo="
         << xtrabackup_encrypt_algo_names[opt_decrypt_algo];
-    if (xtrabackup_encrypt_key) {
-      cmd << " --encrypt-key=" << xtrabackup_encrypt_key;
-    } else {
+    if (xtrabackup_encrypt_key == nullptr) {
```

Comment:
also, I think it is better to unset it after usage.
get old value (getnv()), save it. and restore it after system() call.

Reply:
on second thought, I think, the ENV change done by this xtrabackup process will not be visible to other xtrabackup processes. So pxb exit will auto unset it.

Please verify

Reply:
correct, pxb exists with auto unset.